basque/alde-egin/alde-egin.md

```diff
@@ -1 +1,9 @@
 Pertsona arraroari zure gaztelutik alde egiteko esan diozu eta bera zuri irainka hasi da.
+
+Harresiaren ateak itxi eta gero barrura sartzen zara, eta telebista ikusten hasten zara. Bat-batean, albisteetan ikusten duzun pertsona batek urrezko gaztainak saltzen hari zela gaztelu guztietan.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.